packages/webpack/package.json

```diff
@@ -50,7 +50,6 @@
     "loader-utils": "^2.0.3",
     "mini-css-extract-plugin": "~2.4.7",
     "parse5": "4.0.0",
-    "parse5-html-rewriting-stream": "6.0.1",
```

Comment:
This is no longer needed now that both `index.html` plugins have beeb consolidated into `WriteIndexHtmlPlugin`. Previously we had two different ones for `dev-server` and `build`.